src/queries/survey.js

```diff
@@ -3,24 +3,25 @@ import gql from 'graphql-tag';
 export const DETAILED_SURVEY = gql`
   query Surveys($id: ID!) {
     surveys(ids: [$id]) {
-      ...surveyFields
-    }
-  }
-
-  fragment surveyFields on SurveyPoll {
-    id
-    title
-    questionTitle
-    description
-    date {
-      id
-      dateEnd
-      dateStart
-    }
-    responseOptions {
       id
       title
-      votesCount
+      questionTitle
+      description
+      date {
+        id
+        dateEnd
+        dateStart
+      }
+      responseOptions {
+        id
+        title
+        votesCount
+      }
+      surveyComments {
```

Comment:
what about `comments: surveyComments` here and using `survey.comments` elsewhere?
it was a main server side issue why we could not name it `comments` directly. i use `comments: surveyComments` everywhere (cms code, insomnia, ...)

Reply:
if you already use `comments: surveyComments` in other places i can change it. otherwise i don't mind the name

Reply:
done

Reply:
yes, in cms. in addition i prefer accessing `comments` of a `survey` instead of `surveyComments` of a `survey` for readability. and it is an easy way achieve that. let's do it. thx.